.github/workflows/lint.yml

```diff
@@ -32,16 +32,10 @@ jobs:
       id: ruff
       if: steps.changed-python-files.outputs.all_changed_files != '' && !cancelled()
       run: |
-        ruff check --config=pyproject.toml --show-fixes --output-format=github ${{ steps.changed-python-files.outputs.all_changed_files }}
-
-    - name: Black
-      id: black
-      if: steps.changed-python-files.outputs.all_changed_files != '' && !cancelled()
-      run: |
-        black --config=pyproject.toml --check --diff --verbose ${{ steps.changed-python-files.outputs.all_changed_files }}
+        ruff format --check --config=pyproject.toml .
```

Comment:
we still needed `ruff check .` in our pipeline. it not hte same with `ruff format --check`